solutions/search/query-rules-ui.md

```diff
@@ -0,0 +1,136 @@
+---
+applies_to:
+  stack: ga 9.1
+  serverless: ga
+  elasticsearch:
+---
+
+# Query rules UI
+Use query rules to boost, pin, or exclude specific documents when queries contain certain keywords, phrases, or match defined search patterns.
+The Query rules UI provides a graphical interface to manage these rules without writing API calls or JSON configuration.
+
+The UI enables you to:
+
+- Set keyword triggers and conditions for when rules apply
+- Pin, boost, or exclude specific documents in results
+- Organize rules into rulesets and set execution priority
+- Test rules against sample queries before publishing
+
+## UI vs. API: What's the difference?
+
+The Query Rules UI provides the same functionality as the API with one key difference in how documents are pinned:
+
+* The UI defaults to `docs` for maximum flexibility, but still allows `id`-based pinning for single-index searches through a simplified form.
```

Comment:
This is a little confusing wording to me - IIRC ids were not that well supported in the UI, you can't edit an id based rule for example. Also I don't think the UI allows you to create an empty docs rule, which would behave in the same way as an id rule.